mon-pix/translations/fr.json

```
@@ -503,7 +503,16 @@
         },
         "4": {
           "title": "Que faire en cas de problème ?",
-          "text": "'<span class=\"certification-instructions__text--bold\">'En cas de problème technique'</span>' (par exemple, l'image refuse de s'afficher), suivez ces étapes : '<br><br>' cliquez sur \"'<span class=\"certification-instructions__text--bold\">'Signaler un problème avec la question'</span>'\", puis sur \"Oui, je suis sûr(e)\" et informez votre surveillant."
+          "paragraph": {
+            "strong-text": {
+              "1": "En cas de problème technique",
+              "2": "Signaler un problème avec la question,"
```

Sur la maquette, `Signaler un problème avec la question` est entouré par des guillemets qu'on ne retouve pas sur la RA :

```suggestion
"2": "\"Signaler un problème avec la question\","
```